### PR TITLE
Update bz_nexgen.m3u

### PR DESCRIPTION
**Pull request by ProPontifex**
See title

### CODE REVIEW

**Review by Alstruit — DISMISSED** [review]

File: streams/bz_nexgen.m3u [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+#EXTINF:-1 tvg-id="AXNCentralAmerica.us",AXN
 http://170.254.18.106/AXN/index.m3u8
-#EXTINF:-1 tvg-id="",AMC
+#EXTINF:-1 tvg-id="AMCEast.us",AMC
 http://170.254.18.106/AMC/index.m3u8
-#EXTINF:-1 tvg-id="",Comedy Central
+#EXTINF:-1 tvg-id="ComedyCentralEast.us",Comedy Central
 http://170.254.18.106/COMEDY_CENTRAL/index.m3u8
-#EXTINF:-1 tvg-id="",Lifetime
+#EXTINF:-1 tvg-id="LifetimeEast.us",Lifetime
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:10]: Edited: thanks.

File: streams/bz_nexgen.m3u [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
 http://170.254.18.106/LIFETIME/index.m3u8
-#EXTINF:-1 tvg-id="",HBO Mundi
+#EXTINF:-1 tvg-id="HBOMundiCaribbean.us",HBO Mundi
 http://170.254.18.106/HBO_MUNDI/index.m3u8
-#EXTINF:-1 tvg-id="",HBO Family
+#EXTINF:-1 tvg-id="HBOFamilyEast.us",HBO Family
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:14]: Edited

File: streams/bz_nexgen.m3u [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
 http://170.254.18.106/HBO_FAMILY/index.m3u8
 #EXTINF:-1 tvg-id="",HBO Plus
 http://170.254.18.106/HBO_PLUS/index.m3u8
-#EXTINF:-1 tvg-id="",HBO 2
+#EXTINF:-1 tvg-id="HBO2East.us",HBO 2
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:18]: Edited

File: streams/bz_nexgen.m3u [hunk 4]
```diff
@@ -1,41 +1,41 @@
...
 http://170.254.18.106/HBO2/index.m3u8
-#EXTINF:-1 tvg-id="",HISTORY
+#EXTINF:-1 tvg-id="HistoryEast.us",History
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:20]: Edited

File: streams/bz_nexgen.m3u [hunk 5]
```diff
@@ -1,41 +1,41 @@
...
 http://170.254.18.106/HISTORY/index.m3u8
 #EXTINF:-1 tvg-id="",STAR
 http://170.254.18.106/STAR/index.m3u8
-#EXTINF:-1 tvg-id="",FX
+#EXTINF:-1 tvg-id="FXEast.us",FX
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:24]: Edited

File: streams/bz_nexgen.m3u [hunk 6]
```diff
@@ -1,41 +1,41 @@
...
 http://170.254.18.106/FX/index.m3u8
-#EXTINF:-1 tvg-id="",TNT
+#EXTINF:-1 tvg-id="TNTEast.us",TNT
 http://170.254.18.106/TNT/index.m3u8
-#EXTINF:-1 tvg-id="",HBO Signature
+#EXTINF:-1 tvg-id="HBOSignatureEast.us",HBO Signature
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:28]: Edited

File: streams/bz_nexgen.m3u [hunk 7]
```diff
@@ -1,41 +1,41 @@
...
+#EXTINF:-1 tvg-id="VH1East.us",VH1
 http://170.254.18.106/VH1/index.m3u8
-#EXTINF:-1 tvg-id="",PARAMOUNT NETWORK
+#EXTINF:-1 tvg-id="ParamountNetworkEast.us",Paramount Network
 http://170.254.18.106/PARAMOUNT/index.m3u8
-#EXTINF:-1 tvg-id="",HBO XTREME
+#EXTINF:-1 tvg-id="HBOXtremeLatinAmerica.us",HBO Xtreme
 http://170.254.18.106/HBO_XTREME/index.m3u8
-#EXTINF:-1 tvg-id="",CINEMAX
+#EXTINF:-1 tvg-id="CinemaxEast.us",Cinemax
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:38]: Edited

File: streams/bz_nexgen.m3u [hunk 8]
```diff
@@ -7,25 +7,25 @@ http://170.254.18.106/AXN/index.m3u8
 http://170.254.18.106/AMC/index.m3u8
 #EXTINF:-1 tvg-id="ComedyCentralEast.us",Comedy Central
 http://170.254.18.106/COMEDY_CENTRAL/index.m3u8
-#EXTINF:-1 tvg-id="LifetimeEast.us",Lifetime
+#EXTINF:-1 tvg-id="",Lifetime
 http://170.254.18.106/LIFETIME/index.m3u8
 #EXTINF:-1 tvg-id="HBOMundiCaribbean.us",HBO Mundi
 http://170.254.18.106/HBO_MUNDI/index.m3u8
-#EXTINF:-1 tvg-id="HBOFamilyEast.us",HBO Family
+#EXTINF:-1 tvg-id="HBOLatinAmerica.us",HBO Latin America
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply streams/bz_nexgen.m3u:14]: Edited

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**